src/figvalidationservice/src/test/java/ca/bc/gov/open/pssg/rsbc/dps/figvalidationservice/FigValidationServiceAppTests.java

```diff
@@ -0,0 +1,58 @@
+package ca.bc.gov.open.pssg.rsbc.dps.figvalidationservice;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.test.context.TestPropertySource;
+
+/**
+ * 
+ * @author shaunmillargov
+ *
+ */
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
+@TestPropertySource(locations="classpath:test.properties")
+public class FigValidationServiceAppTests {
+	
+	@LocalServerPort
+	private int port;
+	
+	@Autowired
+	private TestRestTemplate restTemplate;
+	
+	@Autowired 
+	private FigValidationServiceApp figValidationServiceApp; 
+
+	/**
+	 * contextLoaded - Test for fig validation service context.  
```

Comment:
figaro*

Reply:
I think what you were wanting to see was 'fig' as 'Figaro'. If so, it has been changed and committed. Thanks Adam. S.